staging/src/k8s.io/apiserver/pkg/endpoints/installer.go

```diff
@@ -800,6 +801,7 @@ func (a *APIInstaller) registerResourceHandlers(path string, storage rest.Storag
 		}
 		for _, route := range routes {
 			route.Metadata(ROUTE_META_GVK, reqScope.Kind)
+			route.Metadata(ROUTE_META_ACTION, action.Verb)
```

Comment:
seems like this should match the verbs used by authorization and published in discovery